Readme.md

```diff
@@ -155,6 +157,30 @@ Create a new workflow or add the following step to an existing one that's trigge
 
 You have to provide a JSON object for the `scripts` and `styles-css` inputs, where the property names are the relative paths of the projects you want to inspect, and the values become the`nodejsExtensions` properties in the temporarily generated _package.json_ files used for the linting operation. For more information, check out the workflow inputs [here](.github/workflows/lint.yml).
 
+### Simplified configuration
+
+If the files to be linted are located in the _./Assets/Scripts_ and _./Assets/Styles_ subdirectories, then you can use the simplified comma-separated format:
```

Comment:
Actually why don't have use _wwwroot/js_ and _wwwroot/css_ as defaults? It seems to me that the most common scenario will be this, but now we require the most configuration (of which most is copy-pasted) for this scenario. Or alternatively, detect these folders when the simple config is used: e.g. with the `scripts: src/Modules/OrchardCore.Commerce` config, check if there's a _src/Modules/OrchardCore.Commerce/Assets/Scripts_ folder, and if yes, lint that. If not, then check for a _src/Modules/OrchardCore.Commerce/wwwroot/js folder.